devenv/dev-dashboards/panel-table/table_tests_new.json

```diff
@@ -13,41 +16,60 @@
     ]
   },
   "editable": true,
-  "gnetId": null,
+  "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
   "links": [],
+  "liveNow": false,
   "panels": [
     {
-      "datasource": null,
+      "datasource": {
+        "type": "datasource",
+        "uid": "grafana"
+      },
       "gridPos": {
         "h": 1,
         "w": 24,
         "x": 0,
         "y": 0
       },
       "id": 7,
+      "targets": [
+        {
+          "datasource": {
+            "type": "datasource",
+            "uid": "grafana"
+          },
+          "refId": "A"
+        }
+      ],
       "title": "Cell styles",
       "type": "row"
     },
     {
-      "datasource": "gdev-testdata",
+      "datasource": {
+        "type": "testdata",
+        "uid": "jrsS70dMz"
```

Comment:
The uid for gdev-testdata is `PD8C576611E62080A`  

No panels show data for me when I use this branch for this dashboard now (as no datasource can be found)

Reply:
weeeird, mine has different UID. investigating

Reply:
don't see anything that would set `PD8C576611E62080A` specifically when provisioning. Updated dashboard json to use data source name for compatibility